cmake/std/atdm/common/toss3/environment.sh

```diff
@@ -18,6 +18,9 @@ export ATDM_CONFIG_BUILD_COUNT=8
 # NOTE: Above, currently setting CMAKE_JOB_POOL_LINK results in a build
 # failures with Ninja.  See https://gitlab.kitware.com/snl/project-1/issues/60
 
+# We do this twice since sems modules are wacked and we get errors to the screen on a purge
+# The second purge will catch any real errors with purging ...
+module purge &> /dev/null
 module purge
```

Comment:
Perhaps you want to remove this line and change the comment above accordingly.

Reply:
I added that line and made the comment, what don't you like about that line?

Reply:
@bartlettroscoe can you send me exactly what you are doing to get those warnings?

Reply:
Nevermind @bathmatt&mdash;I understand what you were going for now.